backend/sftp/sftp.go

```diff
@@ -41,6 +41,7 @@ var (
 
 	// Flags
 	sftpAskPassword = flags.BoolP("sftp-ask-password", "", false, "Allow asking for SFTP password when needed.")
+	sshPathOverride = flags.StringP("ssh-path-override", "", "", "Override path used by SSH connection.")
```

Comment:
I think this should probably be a config variable rather than a command line flag as it is something you'd always want set for your synology isn't it?